src/carebot-tracker.js

```diff
@@ -304,23 +313,35 @@
             var percent = depthPercent();
             var bucket = percentBucket(percent);
             if (bucket > previousBucket) {
-                var seconds = timer.check().seconds;
-                callback(bucket, seconds);
-            } else {
-                // The user is scrolling back up.
+                callback(bucket, timer.check().seconds);
                 previousBucket = bucket;
```

Comment:
Hey, thanks for review! But i don't understand why we should send events/log percents of page already showed for reader? You know that Google Analytics accept only 500 hits per session? You want log that user "re-read" article? Just want to understand reason ;)